_instead of crashing it's actually just a unlimited loop using the `next` function_

**run using**
```rb
$ ruby crash.rb
```
